big_tests/tests/instrument_event_table.erl

```diff
@@ -0,0 +1,35 @@
+%% @doc This module is injected to the mim node by instrument_helper.
+%% It collects events in an ETS table.
+
+-module(instrument_event_table).
+
+-export([set_up/1, tear_down/0, get_events/2]).
+
+set_up(DeclaredEvents) ->
+    ets_helper:new(?MODULE, [bag]), % repeating measurements are stored only once
+    meck:new(mongoose_instrument_log, [no_link, passthrough]),
+    meck:expect(mongoose_instrument_log, handle_event,
```

Comment:
Hmm... There should be a better way than meck here. If we're already rpc-inserting a module here, and we're creating an ets table in the remote node, we can just rpc-insert a module that implements the `mongoose_instrument` behaviour already and does the ets inserts on its own, entirely bypassing meck.

Reply:
There are many ways without meck, and I tried the following:
- Implementing `mongoose_instrument_*` is not easy, because all instrumentation is set up on startup. You would need to restart `mongooseim`.
- Adding a special list of global handlers. This adds extra code executed with each handler, that is only for tests, so I don't like it.
- Log handler. This required a lot of work to silence the debug logs by reconfiguring existing handlers, and looked really messy.

So I really don't see a cleaner solution.

Reply:
This is unfortunate. Points 2 and 3 aren't good at all from my point of view, so I'd discard them before even trying. Point 1 is the one that sounds best to me, wonder if there's a way to rework how handlers are initialised so that they can be injected dynamically without restarting MIM. I'll make a note for this one, it can be in a separate PR, it's well encapsulated in the instrumentation helper modules.

Reply:
I didn't add this functionality, because it's not needed during normal operation, and I didn't want something just for tests. Also, we want to avoid the situation where configuration (`mongoose_config`) is different from actual setup (an ad-hoc added handler). So it would be something like `dynamic_modules`, but for handlers. It's doable, but do we really need it?